storage/sql/migrations/updates/20190214154800_remove_trs_dependant_tables.sql

```diff
@@ -14,17 +14,10 @@
 
 
 /*
-  DESCRIPTION: Counts duplicate delegates by transactionId.
-
-  PARAMETERS: None
+ * DESCRIPTION: Remove all trs dependant tables into trs asset field (transfer, signatures, delegates, votes, multisignatures, dapps, intransfer, outtransfer)
+ * Note that it will also drop the `full_blocks_list` view as it uses some of these tables
+ *
+ * PARAMETERS: None
 */
 
-WITH duplicates AS
-  (
-    SELECT count(*)
-    FROM delegates
-    GROUP BY "transactionId"
-    HAVING count(*) > 1
-  )
-SELECT count(*)
-FROM duplicates
+DROP TABLE "transfer", "signatures", "delegates", "votes", "multisignatures", "dapps", "intransfer", "outtransfer" CASCADE;
```

Comment:
It should be merged with the migration: `20190204170819_migrate_trs_dependant_tables_to_trs_asset_field.sql`.

Where is the migration of constant `duplicatedDelegates` then?

Reply:
`duplicatedDelegates` was removed because it calculated duplicated `transactionId` in the `delegate` table. As `delegate` table was removed and `id` is uniq in `trs` table, there is no need for that check anymore.

Should I combine `20190214154800_remove_trs_dependant_tables.sql` into `20190204170819_migrate_trs_dependant_tables_to_trs_asset_field.sql`?